policyengine_us/tests/policy/baseline/gov/states/nj/tax/income/credits/eitc/nj_eitc.yaml

```diff
@@ -0,0 +1,45 @@
+- name: New Jersey EITC, fraction of federal EITC (qualifies for federal EITC)
```

Comment:
Since these are unit tests, looking at the Python file, they can be flat based only on `eitc_eligible`, `nj_eitc_eligible`, and `eitc`.